ACKategories-Example/Screens/VC composition/TitleViewController.swift

```diff
@@ -19,7 +20,7 @@ class TitleViewController: UIViewController {
     init(name: String, color: UIColor) {
         self.name = name
         self.color = color
-        super.init(nibName: nil, bundle: nil)
+        super.init(viewModel: 0)
```

Comment:
I don't really like this type of notation, maybe a `typealias NoViewModel = Any` would be nicer? Then `Base.ViewController<Any>` could be written as `Base.ViewController<NoViewModel>` and we could write extension for `Base.ViewController where ViewModelType: NoViewModel` with default implementation`init` (there we can call `viewModel = 0`) without having to call `super.init(viewModel: 0)` from the subclass. Anyway, have not tried this in code, but give it a thought and let me know what you think 😉

Reply:
Now that I think about it `typealias NoViewModel = Any` will not work since all view models will be of type `Any` (and we do not want to provide default implementation for viewControllers with a viewModel), so maybe create an empty type `NoViewModel`?

Reply:
Yep, I'd create a public struct `NoViewModel` with internal `init`, so it cannot be instantiated outside our module and provide default `init()` to `Base.ViewController<NoViewModel>`

Reply:
Good point, I definitely agree 👍 I will do that 💪

Reply:
OK, I did my best. There's no better implementation of your idea AFAIK 😞